tools/autograd/derivatives.yaml

```diff
@@ -138,6 +138,12 @@
 - name: alias(Tensor self)
   self: grad
 
+- name: all(Tensor self)
```

Comment:
does this make any difference?

Reply:
I was hitting type mismatch (`Variable[CPUByteType]` vs CPUByteType) in `TensorIterator` without this change, because codegen is not unwrapping the input Tensor for `all(*)` without this line.

Reply:
I see.  Can you write a comment here that says this is because TensorIterator doesn't work on Variables?  I'm unclear if that part is fixable, you can investigate in a future PR if you want.